scripts/bin/auto-install

```diff
@@ -101,6 +101,15 @@ track_stable() {
     return 1
 }
 
+eject_device() {
+    # Try to copy log file over
+    cp -f "$_log_file" "$_mount_point" || true
+
+    sync
+    umount "$_mount_point"
+    eject "/dev/disk/by-uuid/$(basename "$_mount_point")"
```

Comment:
Remake to `eject_device $WATCH_FILE` with `_disk="$1` and `eject $_disk`, or something like?

Reply:
Is your issue with implicitly passed arguments (`$_mount_point`) or reverse engineering the block device path?

I don't really like implicitly passing args, but my instinct was to have `_mount_point="$1"`, which felt even worse. Other options are to use a different arg for (maybe just `_mount`) or we could also make the function run in a subshell:

```sh
eject_device() (
  _mount_point="$1"
  
  # ...
)
```

Reply:
My thinking was that because we already have a FIFO which holds the path to the partition we can just reuse that

Reply:
> [If a device partition is specified, the whole-disk device is used.](https://www.man7.org/linux/man-pages/man1/eject.1.html)

Nevermind!

Though I think because we have a mounting script we could probably do with moving the eject logic to their and having this script call that script to do the work... Added benefit of merely getting to reuse `$MOUNT_POINT`...

Reply:
I think ejecting is the minimum safety level considering we do write to the disk. I'm thinking that we should have an idea surrounding what cleanup looks like which can be extended in the future (i.e. notifications).

Reply:
That being said, I think I'm satisfied by this PR at this point.